blocks/article-grid/article-grid.css

```diff
@@ -0,0 +1,126 @@
+/* use styles based on live milo libs */
+@import "https://main--milo--adobecom.hlx.live/libs/blocks/section-metadata/section-metadata.css";
```

Comment:
You should be able to use the loadstyle method https://github.com/search?q=repo%3Aadobecom%2Fblog%20loadstyle&type=code or as @rgclayton suggested, have this live in a more general file.

Reply:
Thank you so much! Used `loadStyle` instead of `@import`, thankss